comp/core/flare/helpers/builder.go

```diff
@@ -175,6 +188,12 @@ func (fb *builder) logError(format string, params ...interface{}) error {
 }
 
 func (fb *builder) Logf(format string, params ...interface{}) error {
+	fb.Lock()
```

Comment:
Should we also lock for LogError ?

Reply:
I don't think it's needed for `logError`, worst case it will try to write to a file which is already closed, and silently ignore the error anyway